paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -71,7 +70,6 @@ internal sealed class PaymentSelection : Parcelable {
     @Parcelize
     data class Link(
         val useLinkExpress: Boolean = false,
-        val linkAccount: LinkAccount? = null,
```

Comment:
This is now passed to `FlowController` directly on the activity result, rather than bundled on the selection.